shell/platform/android/io/flutter/embedding/engine/systemchannels/PlatformViewsChannel.java

```diff
@@ -73,6 +73,9 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull MethodChannel.Result
             case "clearFocus":
               clearFocus(call, result);
               break;
+            case "synchronizeToNativeViewHierarchy":
+              synchronizeToNativeViewHierarchy(call, result);
```

Comment:
There's a potential race condition here, e.g. as this is, `synchronizeToNativeViewHierarchy` must be called prior to creating the platform view.

What about we move this flag to the construction params for the platform view?  e.g. where we pass the `viewId`, etc.

Reply:
discussed offline. This might also confused people since this is a global value that affects all platform views.  
Current plan sgtm. 

Note that later we might need to handle the case of a platformView is already displayed, and a method call is made to change the default. For now, I think it's ok to say that we don't support it.